Vagrantfile

```diff
@@ -0,0 +1,40 @@
+Vagrant.configure("2") do |config|
```

Comment:
Looks good, but worth mentioning this in the README

Just a brief "by the way you can run the app in a VM if you prefer, by ..."